src/common/transformations/src/transformations/op_conversions/convert_avgpool_downgrade.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_avgpool_downgrade.hpp"
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/avg_pool.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+ov::pass::ConvertAvgPool14ToAvgPool1::ConvertAvgPool14ToAvgPool1() {
+    MATCHER_SCOPE(ConvertAvgPool14ToAvgPool1);
+
+    const auto avg_pool_v14_pattern = pattern::wrap_type<ov::op::v14::AvgPool>();
+
+    const matcher_pass_callback callback = [](pattern::Matcher& m) {
+        const auto avg_pool_v14 = std::dynamic_pointer_cast<ov::op::v14::AvgPool>(m.get_match_root());
```

Comment:
from the code, it looks like we always can donwgrade  avg_pool v14 to v1.
Are there any cases where we can't downgrade?

should we "return false" for these cases to prevent this downgrade?